src/background/index.js

```diff
@@ -0,0 +1,44 @@
+import APIService from "../popup/APIService";
+import SLStorage from "../popup/SLStorage";
+
+import { handleNewRandomAlias } from "./create-alias";
+import { handleOnClickContextMenu } from "./context-menu";
+
+global.browser = require("webextension-polyfill");
+global.isBackgroundJS = true;
+
+/**
+ * Get app settings
+ */
+async function handerGetAppSettings() {
```

Comment:
a small type here, `hander` should be `handler` :)

Reply:
fixed on 493574a0296cbab6de218ea8de6cda8dbbb09ae1